bioptim/dynamics/dynamics_functions.py

```diff
@@ -210,12 +238,12 @@ def torque_activations_driven(states: MX.sym, controls: MX.sym, parameters: MX.s
 
         dq = horzcat(*[dq for _ in range(ddq.shape[1])])
 
-        return vertcat(dq, ddq)
+        return vertcat(dq, ddq), MX(nlp.states.shape, ddq.shape[1])
```

Comment:
Does this actually work, or the MX is a place holder? If so, please make sure we raise an `NotImplementedError` somewhere so noone tries to use this dynamic function

Reply:
The flag is hidden in the collocation integrator, it allows to switch from the new implicit constraint to the older one when MX=0 is detected.